Services/JobRunner/InstanceExportDescriptorProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+
+using System.Collections.Generic;
+using System.Composition.Hosting.Core;
+
+namespace JobRunner
+{
+    /// <summary>
+    /// Custom export descriptor provider class to add existing instances to MEF container
+    /// </summary>
+    public class InstanceExportDescriptorProvider<TValue> : ExportDescriptorProvider
```

Comment:
There is already an existing implementation for this, can you just repurpose the existing one instead of creating a dup class?